cartesi-rollups_versioned_docs/version-1.5/rollups-apis/graphql/objects/completion-status.md

```diff
@@ -0,0 +1,34 @@
+---
+id: completion-status
+title: CompletionStatus
+hide_table_of_contents: false
+---
+
+Enum representing the possible statuses of an input's processing.
+
+```graphql
+enum CompletionStatus {
+  Unprocessed
+  Accepted
+  Rejected
+  Exception
+  MachineHalted
+  CycleLimitExceeded
+  TimeLimitExceeded
+  PayloadLengthLimitExceeded
+}
+```
+
+## Values
+
+| Name | Description |
+| ---- | ----------- |
+| `Unprocessed` | The input has not been processed yet. |
+| `Accepted` | The input was accepted and processed successfully. |
+| `Rejected` | The input was rejected and not processed. |
```

Comment:
This one means "it was processed and the results were rejected (ie: state reverted)"

Reply:
Resolved.